test/core/memory_usage/BUILD

```diff
@@ -31,11 +32,11 @@ grpc_cc_binary(
     ],
 )
 
-grpc_cc_binary(
+cc_library(
```

Comment:
No, please use `grpc_cc_library`. If you require copts, you need to make this a bit more global.

Reply:
Done.